compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -606,6 +606,7 @@ class ValuePropagation : public TR::Optimization
    void transformUnknownTypeArrayCopy(TR_TreeTopWrtBarFlag *);
    void transformReferenceArrayCopy(TR_TreeTopWrtBarFlag *);
    void transformReferenceArrayCopyWithoutCreatingStoreTrees(TR_TreeTopWrtBarFlag *arrayTree, TR::SymbolReference *srcObjRef, TR::SymbolReference *dstObjRef, TR::SymbolReference *srcRef, TR::SymbolReference *dstRef, TR::SymbolReference *lenRef);
+   virtual void transformRecognizedMethod(TR::Node *node) { }
```

Comment:
Documentation is added in the new commit which will be merged with other commits before deliver.